poetry/core/json/schemas/poetry-schema.json

```diff
@@ -92,7 +92,42 @@
         },
         "include": {
             "type": "array",
-            "description": "A list of files and folders to include."
+            "description": "A list of files and folders to include.",
+            "items": {
+                "anyOf": [
+                    {
+                        "type": "string"
+                    },
+                    {
+                        "type": "object",
+                        "description": "Information about where the file or folder resides.",
+                        "additionalProperties": false,
+                        "required": [
+                            "include"
+                        ],
+                        "properties": {
+                            "include": {
+                                "type": "string",
+                                "description": "What to include in the package."
+                            },
+                            "format": {
```

Comment:
we should probably move this to defintions section with as something like `package-format` and restrict values to what we support - `sdist`, `wheel` etc. Might need to check of 'bdist' is supported as well. Also, might be good to have descriptions for strings.

Reply:
Not sure if `bdist` is necessary since it's just used for [extracting metadata](https://github.com/python-poetry/core/blob/master/poetry/core/_vendor/pkginfo/tests/test_bdist.py). Or do you mean that we should treat `wheel == bdist`?